core/src/smartcontracts/isi/mod.rs

```diff
@@ -483,11 +292,11 @@ mod tests {
         let account_id = AccountId::from_str("alice@wonderland")?;
         let (public_key, _) = KeyPair::generate()?.into();
         let asset_definition_id = AssetDefinitionId::from_str("rose#wonderland")?;
-        RegisterExpr::new(Domain::new(DomainId::from_str("wonderland")?))
+        RegisterBox::domain(Domain::new(DomainId::from_str("wonderland")?))
```

Comment:
```suggestion
        RegisterBox::domain("wonderland")?
```

doesn't this make more sense? We need to simplify API. The same goes for others